case_studies/strong_lensing/galsim-random.yaml

```diff
@@ -0,0 +1,79 @@
+
+variables :
+  output_directory : data/images
+  n1 : 2.5
+  half_light_radius : 2
+  flux1 : .3
+  n2 : 1.5
+  scale_radius : 1
+  flux2 : .7
+  q : .7
+  beta : 23
+
+# Define the galaxy profile
+gal :
+    type : Sum
+    items :
+        -   type : Sersic
+            n : '@variables.n1'
+            half_light_radius : '@variables.half_light_radius'
+            flux : '@variables.flux1'
+        -   type : Sersic
+            n : '@variables.n2'
+            scale_radius : '@variables.scale_radius'
+            flux : '@variables.flux2'
+  
+    shear :
+        type : QBeta
+        q : '@variables.q'
+        beta : '@variables.beta'
+
+    flux : 1.e6
+
+# Define the PSF profile
+psf :
+    type : Convolution
+    items :
+        -
+            type : Kolmogorov
+            fwhm : {type: Random, min: 1.5, max: 2.5}
+            ellip:
+                type : EBeta
+                e : {type: Random, min: 0.0, max: 0.2}
+
+                beta :
+                    type : Random
+                    
+        -
+            type : OpticalPSF
+            lam_over_diam : 0.041253
+            defocus : {type: Random, min: 0, max: 0.6}
+            astig1 : {type: Random, min: -0.3, max: 0.3}
+            astig2 : {type: Random, min: -0.3, max: 0.3}
+            coma1 : {type: Random, min: -0.3, max: 0.7}
+            coma2 : {type: Random, min: -0.3, max: 0.3}
+            obscuration : {type: Random, min: 0.2, max: 0.4}
+
+# Define some other information about the images
+image :
+    pixel_scale : 0.23
```

Comment:
This file has pixel scale as 0.26 arcsecond:
https://github.com/GalSim-developers/GalSim/blob/5c8ff6da1c077062fc4aabd70a164b3a208e76de/examples/des/blendset.yaml#L105

Where are your numbers coming from? Could it be different for coadds?

Reply:
I was just trying random numbers, wasn't much logic behind choosing the actual value.